/home/runner/work/KDT_Y_BE_Toy_Project2_DEV/KDT_Y_BE_Toy_Project2_DEV/src/test/java/com/fastcampus/toyproject/domain/itinerary/util/ItineraryOrderUtilTest.java:47: error: incompatible types: List<Itinerary> cannot be converted to List<Integer>
            ItineraryOrderUtil.validateItinerariesOrder(testList1);

테스트 쪽 확인해보시면 될거 같아요

--- a/src/main/java/com/fastcampus/toyproject/domain/itinerary/dto/ItineraryResponseFactory.java
+++ b/src/main/java/com/fastcampus/toyproject/domain/itinerary/dto/ItineraryResponseFactory.java
@@ -9,6 +9,10 @@ import java.util.HashMap;
 import java.util.Map;
 import java.util.function.Function;
 
+/**
+ * 여정의 타입이 들어오면 그에 따라 response를 반환해주는
+ * 팩토리 패턴을 적용한 클래스
+ */
 public class ItineraryResponseFactory {
     private final static Map<ItineraryType, Function<Itinerary, ItineraryResponse>> map = new HashMap<>();
 

--- a/src/main/java/com/fastcampus/toyproject/domain/itinerary/entity/ItineraryFactory.java
+++ b/src/main/java/com/fastcampus/toyproject/domain/itinerary/entity/ItineraryFactory.java
@@ -7,6 +7,10 @@ import java.util.HashMap;
 import java.util.Map;
 import java.util.function.BiFunction;
 
+/**
+ * request의 여정 타입이 들어오면 그에 따라 entity를 반환해주는
+ * 팩토리 패턴을 적용한 클래스
+ */
 public class ItineraryFactory {
     private final static Map<ItineraryType, BiFunction<Trip,
             ItineraryRequest, Itinerary>> map = new HashMap<>();

--- a/src/test/java/com/fastcampus/toyproject/domain/itinerary/util/ItineraryOrderUtilTest.java
+++ b/src/test/java/com/fastcampus/toyproject/domain/itinerary/util/ItineraryOrderUtilTest.java
@@ -16,7 +16,7 @@ import static com.fastcampus.toyproject.domain.itinerary.exception.ItineraryExce
 class ItineraryOrderUtilTest {
 
     LocalDateTime testStartDate, testEndDate;
-    List<Itinerary> testList1, testList2;
+    List<Integer> testList1, testList2;
 
     public ItineraryOrderUtilTest() {
         this.testList1 = new ArrayList<>();

